### PR TITLE
Tribute Page - Dr. Norman Borlaug: The man who saved a billion lives

### PR DESCRIPTION
**Pull request by fernandorpm**
# Tribute Page - Dr. Norman Borlaug: The man who saved a billion lives

## To add
- index.html
- css/style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  <h1 id="title">Dr. Norman Borlaug</h1>
+  <p id="tribute-info">The man who saved a billion lives</p>
+
+  <div id="img-div">
+    <img id="image" src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg" alt="" width="55%">
+    <div class="page-space"></div>
+    <caption id="img-caption">This is a beautifully taken photo!</caption>
+    <div class="page-space"></div>
+  </div>
```
Comment (fernandorpm) [inline index.html:21]: - [ ] Wrap it into a `section`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      <li><b>2009</b> - dies at the age of 95.</li>
+    </ul>
+
+    <br>
+    <i>
+      <p class="quotation">"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."</p>
+      <p class="quotation">-- Indian Prime Minister Manmohan Singh</p>
+    </i>
+
+  </div>
```
Comment (fernandorpm) [inline index.html:63]: - [ ] Wrap it into a `section`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+
+
+<br>
+
+  <h3>If you have time, you should read more about this incredible human being on his
+    <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia Entry</a>.</h2>
```
Comment (fernandorpm) [inline index.html:70]: - [ ] You closed the `h3` tag wrongly with `h2`

Comment (fernandorpm) [inline index.html:70]: - [ ] Wrap it into a `section`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      <li><b>1962</b> - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help
+        mitigate mass starvation due to a rapidly expanding population</li>
+      <li><b>1970</b> - receives the Nobel Peace Prize</li>
+      <li><b>1983</b> - helps seven African countries dramatically increase their maize and sorghum yields</li>
+      <li><b>1984</b> - becomes a distinguished professor at Texas A&M University</li>
+      <li><b>2005</b> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are
+        the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous
+        because "we've been genetically modifying plants and animals for a long time. Long before we called it science,
+        people were selecting the best breeds."</li>
+      <li><b>2009</b> - dies at the age of 95.</li>
```
Comment (fernandorpm) [inline index.html:54]: - [ ] Break the lines after the `<li>` and before `</li>`

**Review by fernandorpm — COMMENTED** [review]
# :warning: Changes Required on this Pull Request! :warning:
Hello there!

You have done a great job so far, and are one step closer to becoming the best developer ever! :sunglasses:



## Required
> These changes HAVE TO be completed in order for your project to be approved. This means that the project doesn't attend to Microverse's Standard yet. I know that it can be frustrating, but they should be seen as part of your path to become an even better Developer!

- Check the comments under review marked with the tag **[REQUIRED CHANGES]**.
- Please be sure to update your README with the project's information. [Remember that this is the expected structure for a Microverse Project's README](https://github.com/microverseinc/readme-template).

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Just keep in mind that Microverse has a standard for the project requirements, to guarantee that you will be able to become a great developer, and both you and I have to follow them!
___

## Optional
> These changes are NOT mandatory. But if you want to sky-rocket :rocket: your skills and your projects, go for it! :sunglasses:

- Check the comments below marked with the tag **[OPTIONAL]**.
___
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Cheers and Happy Coding! :heart:

> *As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

File: index.html (hunk 1 above)
Comment (mikethreels) [reply index.html:21]: Using sections is important for semantics and it is great that you pointed this out for the developer. However, I would suggest that you make the feedback more polite and not command. Also, it would be better to give a reason as to why the developer should wrap it into a `section`.

File: index.html (hunk 4 above)
Comment (mikethreels) [reply index.html:54]: Although this does make the code more readable it is not a requirement, please make sure that you mention it at the beginning of the comment in case it is [OPTIONAL]. please also include the reason as to why these changes should be made.

File: index.html (hunk 2 above)
Comment (mikethreels) [reply index.html:63]: please have a look at my previous comment, although you're right I would rephrase this as follows: Please make sure you use semantic HTML. Making use of the `section` tag will improve the information flow in your webpage and the accessibility for people using screenreaders for example. you can refer to [this MDN web docs](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) article for more information regarding semantics.

File: index.html (hunk 3 above)
Comment (mikethreels) [reply index.html:70]: Opening and closing with the same tag is important however, it's good to substantiate why it should be done, explaining it makes the codebase look unprofessional and hard to read for developers working on it in the future.

Comment (mikethreels) [reply index.html:70]: please refer to my previous 2 comments regarding this topic.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Fernando, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
